src/cmd.rs

```diff
@@ -427,6 +427,13 @@ pub enum AdminSub {
         #[structopt(subcommand)]
         target_type: DeleteSub,
     },
+
+    /// [WIP] Create or update DynamoDB tables based on CloudFormation template files (.cfn.yml).
+    #[structopt(setting(structopt::clap::AppSettings::Hidden))]
```

Comment:
`hidden = true` does not work

Failure:

ref:
https://docs.rs/clap/2.34.0/clap/struct.Arg.html#method.hidden
https://github.com/TeXitoi/structopt/issues/466
```rs
$ git diff
diff --git a/src/cmd.rs b/src/cmd.rs
index b9235a1..ba31311 100644
--- a/src/cmd.rs
+++ b/src/cmd.rs
@@ -429,7 +429,7 @@ pub enum AdminSub {
     },
 
     /// [WIP] Create or update DynamoDB tables based on CloudFormation template files (.cfn.yml).
-    #[structopt()]
+    #[structopt(hidden = true)]
```

```zsh
$ cargo build
   Compiling dynein v0.2.1 (/Users/herotaka/Desktop/dynein)
error[E0599]: no method named `hidden` found for struct `App` in the current scope
   --> src/cmd.rs:432:17
    |
432 |     #[structopt(hidden = true)]
    |                 ^^^^^^ method not found in `App<'_, '_>`
```

Success:

ref: 
https://docs.rs/clap/2.34.0/clap/enum.AppSettings.html#variant.Hidden
https://github.com/TeXitoi/structopt/issues/444
```rs
$ git diff
diff --git a/src/cmd.rs b/src/cmd.rs
index b9235a1..ed52d50 100644
--- a/src/cmd.rs
+++ b/src/cmd.rs
@@ -429,7 +429,7 @@ pub enum AdminSub {
     },
 
     /// [WIP] Create or update DynamoDB tables based on CloudFormation template files (.cfn.yml).
-    #[structopt()]
+    #[structopt(setting(structopt::clap::AppSettings::Hidden))]
```

Reply:
Thank you for pointing out. I understood that this is due to structopt require us to specify the different option for subcomand.